examples/sample-app/README.md

```diff
@@ -118,7 +119,7 @@ This section covers how to perform all the steps of building, deploying, and upd
 
        **VAGRANT USERS**: Instead of the above command, use
 
-        $ sudo /data/src/github.com/openshift/origin/_output/local/bin/linux/amd64/openshift start --public-master=localhost --volume-dir=</absolute/path> &> openshift.log &
+        $ sudo `which openshift` start --public-master=localhost --volume-dir=</absolute/path> &> openshift.log &
```

Comment:
We always need to specify a full path to the binary because we are using `sudo` here and `root` doesn't have our binaries in his `$PATH`.

Also, I'm using `which` here to make command a bit shorter and be consistent with `CONTRIBUTING.adoc`